circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
```

Comment:
Yea, ssh'ing into ci is nice, and afik travis doesn't offer it. travis is still fairly defacto standard so would prefer to stick to (just) that for now

was this just more convenient or were you not able to reproduce the issue (e.g. in a local vm)?